.travis.yml

```diff
@@ -31,6 +31,28 @@ matrix:
         - echo '{"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
         - sudo service docker restart
 
+    - rvm: 2.5.1
+      env:
+        - BEAKER_PUPPET_COLLECTION=puppet5
+        - BEAKER_setfile=centos7-64{hostname=centos7-64.example.com,image=centos:7.6.1810}
```

Comment:
I think this is being interpreted by bash as something to expand, causing things to fail.